client/src/features/ProjectPageV2/ProjectPageContent/ProjectInformation/ProjectInformationButton.tsx

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright 2024 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cx from "classnames";
+import { useCallback, useState } from "react";
+
+import { DropdownItem } from "reactstrap";
+
+import { ButtonWithMenuV2 } from "../../../../components/buttons/Button";
+import BootstrapCopyIcon from "../../../../components/icons/BootstrapCopyIcon";
+import useLegacySelector from "../../../../utils/customHooks/useLegacySelector.hook";
+
+import type { Project } from "../../../projectsV2/api/projectV2.api";
+import { useGetUserQuery } from "../../../usersV2/api/users.api";
+
+import useProjectPermissions from "../../utils/useProjectPermissions.hook";
+import ProjectCopyModal from "../../ProjectPageHeader/ProjectCopyModal";
+
+export default function ProjectInformationButton({
+  project,
+}: {
+  userPermissions: ReturnType<typeof useProjectPermissions>;
+  project: Project;
+}) {
+  const { data: currentUser } = useGetUserQuery();
+  const [isCopyModalOpen, setCopyModalOpen] = useState(false);
+  const toggleCopyModal = useCallback(() => {
+    setCopyModalOpen((open) => !open);
+  }, []);
+  const userLogged = useLegacySelector<boolean>(
+    (state) => state.stateModel.user.logged
+  );
+  return (
+    <>
+      <ButtonWithMenuV2
+        color="outline-primary"
+        default={null}
+        preventPropagation
+        size="sm"
+      >
+        <DropdownItem
+          data-cy="project-copy-menu-item"
+          onClick={toggleCopyModal}
+          disabled={!userLogged}
+        >
+          <BootstrapCopyIcon className={cx("bi")} />
+          <span className={cx("ms-2")}>Copy project</span>
+        </DropdownItem>
+      </ButtonWithMenuV2>
+      {isCopyModalOpen && userLogged && (
```

Comment:
This will make the modal animation bug out.

```suggestion
      {userLogged && (
```

Reply:
Fixed in 9b0918ba71221c2bfdfae80a4fe595951ab0018d